include/aws/io/private/pki_utils.h

```diff
@@ -49,6 +49,16 @@ AWS_IO_API int aws_read_and_decode_pem_file_to_buffer_list(
     const char *filename,
     struct aws_array_list *cert_chain_or_key);
 
+/**
+ * Decodes a PEM file to DER format and adds the results to 'der_buffer' if successful.
```

Comment:
Nit: what does adds mean in this context? its not adding to the buffer, but reinitializing it.

Reply:
The function is deprecated.